lib/v27/OC.d.ts

```diff
@@ -0,0 +1,46 @@
+import { Route } from "vue-router";
+
+declare global {
+    namespace Nextcloud.v27 {
+        interface FilesRouter {
+            /**
+            * Trigger a route change on the files app
+            * 
+            * @param path the url path, eg: '/trashbin?dir=/Deleted'
+            * @param replace replace the current history (default false)
+            * @see https://router.vuejs.org/guide/essentials/navigation.html#navigate-to-a-different-location
+            */
+            goTo(path: string, replace?: boolean): Promise<Route>;
+        
+            /**
+             * Trigger a route change on the files App
+             *
+             * @param name the route name
+             * @param params the route parameters
+             * @param query the url query parameters
+             * @param replace replace the current history
+             * @see https://router.vuejs.org/guide/essentials/navigation.html#navigate-to-a-different-location
+             */
+            goToRoute(
+                name?: string,
+                params?: Record<string, string>,
+                query?: Record<string, string | (string | null)[] | null | undefined>,
+                replace?: boolean,
+            ): Promise<Route>;
+        }
+
+        interface OC extends Nextcloud.v26.OC {
+
+        }
+
+        interface OCP extends Nextcloud.v26.OCP {
+            Files: {
+                Router: FilesRouter
+            }
+        }
+
+        interface WindowWithGlobals extends Nextcloud.Common.DayMonthConstants, Window {
```

Comment:
What is this? :thinking:

Reply:
[The same thing we do every release](https://www.youtube.com/watch?v=XJYmyYzuTa8)

https://github.com/nextcloud/nextcloud-typings/blob/84288bfd7c8026f9c2247401cff18a646e4f70e4/lib/v26/OC.d.ts#L11

Reply:
`Nextcloud.Common.DayMonthConstants` seemed like an odd variable name :)